app/views/DR/HomeScreen/index.js

```diff
@@ -47,14 +47,20 @@ export default class HomeScreen extends Component {
     const oldCases = [cases, deaths, recovered, todayCases];
 
     const newCases = oldCases.map(number => {
-      if (number < 1e3) return number;
-      else if (number >= 1e3 && number < 1e5)
-        return number.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
-      else if (number >= 1e5 && number < 1e6)
-        return +(number / 1e3).toFixed(1) + 'K';
-      else return +(number / 1e6).toFixed(1) + 'M';
-    });
+      switch (true) {
```

Comment:
change switch condition from true for example: to number > 0 checkings if the number is positive that way bypassing possible errors